pkg/activator/net/throttler_test.go

```diff
@@ -176,15 +175,23 @@ func TestThrottlerUpdateCapacity(t *testing.T) {
 		numActivators:        2,
 		containerConcurrency: 10,
 		podTrackers:          makeTrackers(3, 10),
-		want:                 15,
+		want:                 20,
```

Comment:
can we add a test with index: 1 to highlight we expect that to be 10

Reply:
Sure! The test case below is the one you are looking for.